### PR TITLE
Use Heroku new review app

### PR DESCRIPTION
**Pull request by patjouk**
- [x] update `app.json`: review apps have their own global setting pages now: sensitive env vars need to be added there directly,
- [x] update `review_app_admin` management command: [we need to replace](https://devcenter.heroku.com/articles/github-integration-review-apps#injected-environment-variables) `HEROKU_APP_NAME` by `HEROKU_PR_NUMBER`.
- [x] Post Thunderbird RA links on a different channel.
- [x] Update and write new docs (how to add env vars, etc)
- [ ] Move `wagtail donate` to the new system because as soon as this PR is merged, they will stop to work.
- [x] Check the permissions changes: https://devcenter.heroku.com/articles/github-integration-review-apps#permission-changes
- [ ] Remove unused env var from staging: github token and review app slack webhook.

### CODE REVIEW

**Comment by patjouk** [comment]
The post deploy was hanging because of a platform issue on Heroku.

It's working, I just need to update our review app script, update the `donate-wagtail` pipeline, and update the doc.

**Comment by patjouk** [comment]
Things to consider as a follow up PR: adapt the review app fake data generator to create thunderbird content.

**Comment by patjouk** [comment]
I'm nearly done with this: I still need to rotate credentials for the thunderbird pipeline, but migrating the donate pipeline will require some downtime. I'll probably do that tomorrow when I'm the only one around.

File: donate/core/management/commands/review_app_admin.py [hunk 1]
```diff
@@ -32,37 +31,53 @@ def handle(self, *args, **options):
...
-            # Get PR's title from Github
-            token = settings.GITHUB_TOKEN
-            org = 'mozilla'
-            repo = 'donate-wagtail'
-            r = requests.get(f'https://api.github.com/repos/{org}/{repo}/pulls/{pr_number}&access_token={token}')
-            try:
-                pr_title = ': ' + r.json()['title']
-            except KeyError:
-                pr_title = ''
+            # As 01/2020 we can only get the PR number if the review app was automatically created
```
Comment (cadecairos) [inline donate/core/management/commands/review_app_admin.py]:
```suggestion
            # As of 01/2020 we can only get the PR number if the review app was automatically created
```

**Comment by patjouk** [comment]
Since they're a bit of manual work to do to make that PR work, I'll merge it myself tomorrow.

**Pull request merged**